src/Chapter1.hs

```diff
@@ -632,8 +654,13 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
-
+sumLast2 :: Int -> Int 
+sumLast2 n = firstLast n + secondLast n
+    where
+        firstLast :: Int -> Int
+        firstLast x = mod (abs x) 10
```

Comment:
Great!
You can also notice that it is actually the `lastDigit` function from the previous task 👍🏼